svgr.config.js

```diff
@@ -79,6 +106,8 @@ export default ${reactComponentName};
 
 module.exports = {
   icon: false,
+  expandProps: 'end',
```

Comment:
I suppose that's true. I haven't seen any effects either so it should be safe to remove it. [8d72088](https://github.com/commercetools/ui-kit/pull/2524/commits/8d72088ddc79425a18cec795b798b60a20b045f0)
Unless @CarlosCortizasCT thinks otherwise?

Reply:
Maybe this was something I added when I was playing with the config at the very beginning.

I think we don't actually needed now ([reference](https://commercetools.atlassian.net/browse/SHIELD-782)).